### PR TITLE
rm ghost and lcnet, add MobileNetV3_small_x1_0

### PR DESCRIPTION
**Pull request by WenmuZhou**

### CODE REVIEW

File: configs/centernet/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-English | [简体中文](README_cn.md)
+**English | [简体中文](README_cn.md)
```
Comment (littletomatodonkey) [inline configs/centernet/README.md]: **可以去掉

File: configs/centernet/README.md [hunk 2]
```diff
@@ -29,4 +33,4 @@ English | [简体中文](README_cn.md)
   journal={arXiv preprint arXiv:1904.07850},
   year={2019}
 }
-```
+```**
```
Comment (yghstill) [inline configs/centernet/README.md]: 末尾的**也去掉吧

File: configs/centernet/README.md (hunk 1 above)
Comment (WenmuZhou) [reply configs/centernet/README.md]: done，tks

File: configs/centernet/README.md (hunk 2 above)
Comment (WenmuZhou) [reply configs/centernet/README.md]: done，tks

**Review by littletomatodonkey — APPROVED** [review]

**Review by yghstill — APPROVED** [review]

**Pull request merged**